code/game/gamemodes/game_mode.dm

```diff
@@ -558,7 +558,7 @@ var/global/list/additional_antag_types = list()
 
 	for(var/mob/living/carbon/human/character in player_list)
 		if(character.client)
-			if(character.client.prefs.economic_status == ECONOMICALLY_DESTITUTE) // Discrimination.
+			if(character.client.prefs.economic_status == ECONOMICALLY_DESTITUTE || ECONOMICALLY_RUINED) // Discrimination.
```

Comment:
I believe this does not work, the logic is wrong, such that it will either always or never trigger depending on the precedence, the correct form of this check is:

```suggestion
			if((character.client.prefs.economic_status == ECONOMICALLY_DESTITUTE) || (character.client.prefs.economic_status == ECONOMICALLY_RUINED)) // Discrimination.
```

There probably also exist a possibility to do more esotheric operations, such as initializing a list with the two defines and checking with `in`. Avoid it, the set of conditions isn't that long to warrant it here.

Reply:
I tried 
`if(character.client.prefs.economic_status == ECONOMICALLY_DESTITUTE||character.client.prefs.economic_status == ECONOMICALLY_RUINED)`
in a little javascript operator demo thing at first. It seemed to work fine, but everyone here seems more knowledgeable in this stuff than me.